repository-data/application/src/main/resources/hcm-config/namespaces/hippostd/html.yaml

```diff
@@ -1,7 +1,7 @@
 definitions:
   config:
     /hippo:namespaces/hippostd/html/editor:templates/_default_:
-      ckeditor.config.appended.json: '{ removeButtons: "PickImage,Table", stylesSet:
+      ckeditor.config.appended.json: '{ removeButtons: "PickImage,Table,Source", stylesSet:
```

Comment:
@FranciscoRuizHEE - do we have a ticket to remove the `source` button across all rich text box instances ?

Reply:
Reverted as discussed with @sankar-shunmuga (as part of 4cc8d9b387da01aa3d8f2ea240bfb293338c9bb5)